examples/SDG1.ipynb

```diff
@@ -0,0 +1,1336 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "I_IHQTO8xXBn"
+      },
+      "source": [
+        "# Synthetic Data generation (Part 1)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "VBoxtnxVdTWZ"
+      },
+      "source": [
+        "\n",
+        "Synthetic data generation using large language models (LLMs) offers a powerful solution to a commonly faced problem: the availability of high-quality, diverse, and privacy-compliant data. This could be used in a number of scenarios such as training a data science  machine learning model (SVMs, decision trees, KNN's), finetuning a different GPT model on the data, as a solution to the coldstart problem, helping build compelling demos/apps with realistic data, scenario testing etc.\n",
+        "\n",
+        "There are a number of key drivers which may see you wanting to leverage synthetic data. For one, human data may have privacy restrictions and/or identifiable data within it which we do not want to be used. Synthetic data can be much more structured and therefore easier to manipulate than real data. In domains where data is sparse or data of certain categories is sparse we may want to augment the data. When dealing with imbalanced datasets or datasets which lack diversity, we may want to create data to improve the richness of our datasets.\n",
+        "\n",
+        "Unlike traditional data augmentation or manual data creation methods, using LLMs allows for the generation of rich, nuanced, and contextually relevant datasets that can significantly enhance it's usefulness to enterprises and developers.\n",
+        "\n",
+        "We split this tutorial into 2 parts. In this cookbook, we delve into the practicalities of generating synthetic data using LLMs, with the following agenda:\n",
```

Comment:
what is the 2nd cookbook focused on?

Reply:
The second cookbook will outline prompt engineering techniques for generating synthetic textual data. (shared in internal doc)